integration_tests/src/main/python/udf_cudf_test.py

```diff
@@ -54,64 +59,68 @@ def _assert_cpu_gpu(cpu_func, gpu_func, cpu_conf={}, gpu_conf={}, is_sort=False)
         assert cpu_ret.sort() == gpu_ret.sort()
     else:
         assert cpu_ret == gpu_ret
-        
 
+
+# ======= Test Scalar =======
 @pandas_udf('int')
 def _plus_one_cpu_func(v: pd.Series) -> pd.Series:
     return v + 1
 
+
 @pandas_udf('int')
 def _plus_one_gpu_func(v: pd.Series) -> pd.Series:
     import cudf
-    gpu_serises = cudf.Series(v)
-    gpu_serises = gpu_serises + 1
-    return gpu_serises.to_pandas()
+    gpu_series = cudf.Series(v)
+    gpu_series = gpu_series + 1
+    return gpu_series.to_pandas()
+
 
-@allow_non_gpu(any=True)
-@pytest.mark.skip("exception in docker: OSError: Invalid IPC stream: negative continuation token, skip for now")
 @cudf_udf
-def test_with_column():
+def test_with_column(enable_cudf_udf):
     def cpu_run(spark):
         df = _create_df(spark) 
         return df.withColumn("v1", _plus_one_cpu_func(df.v)).collect()
 
     def gpu_run(spark):
         df = _create_df(spark)
         return df.withColumn("v1", _plus_one_gpu_func(df.v)).collect()
-    
+
     _assert_cpu_gpu(cpu_run, gpu_run, gpu_conf=_conf)
 
-@allow_non_gpu(any=True)
-@pytest.mark.skip("exception in docker: OSError: Invalid IPC stream: negative continuation token, skip for now")
+
 @cudf_udf
-def test_sql():
+def test_sql(enable_cudf_udf):
     def cpu_run(spark):
         _ = spark.udf.register("add_one_cpu", _plus_one_cpu_func)
-        return spark.sql("SELECT add_one_cpu(id) FROM range(3)").collect()
+        _create_df(spark).createOrReplaceTempView("test_table_cpu")
+        return spark.sql("SELECT add_one_cpu(id) FROM test_table_cpu").collect()
+
     def gpu_run(spark):
         _ = spark.udf.register("add_one_gpu", _plus_one_gpu_func)
-        return spark.sql("SELECT add_one_gpu(id) FROM range(3)").collect()
-    
+        _create_df(spark).createOrReplaceTempView("test_table_gpu")
+        return spark.sql("SELECT add_one_gpu(id) FROM test_table_gpu").collect()
+
     _assert_cpu_gpu(cpu_run, gpu_run, gpu_conf=_conf)
 
 
+# ======= Test Scalar Iterator =======
 @pandas_udf("long")
 def _plus_one_cpu_iter_func(iterator: Iterator[pd.Series]) -> Iterator[pd.Series]:
     for s in iterator:
         yield s + 1
 
+
```

Comment:
remove extra newline

Reply:
Actually two lines between two functions is recommended by the IntelliJ IDE.